client/src/components/traceView.tsx

```diff
@@ -55,11 +55,22 @@ cytoscape.use(coseBilkent);
   
           // console.log("EVT", cy);
   
-          cy.on("tap", "node", evt => {
+          cy.on("dblclick", "node", evt => {
             var node = evt.target;
             console.log("EVT", evt);
             console.log("TARGET", node.data());
             console.log("TARGET TYPE", typeof node[0]);
+            cy.fit( cy.$(':selected'), 50 );
+            setTimeout( function(){
+              cy.panBy({
+                x: -300,
+                y: 0
+              })
+            }, 10)
+            setTimeout( function(){
+              cy.$('').unselect();
+              cy.fit(cy.$(''),50);
+            }, 5000 );
```

Comment:
I mentioned this elsewhere and made a ticket for it on Notion, but I think we can combine create a reusable component for clusterView and traceView files and just pass props down for the differences in the two.

Not a top priority but we should probably squeeze it in before we wrap things up. Definitive best practice.